lightning_pose/utils/scripts.py

```diff
@@ -439,6 +448,97 @@ def get_model(
     return model
 
 
+@typechecked
+def compute_metrics(
+    cfg: DictConfig,
+    preds_file: str,
+    data_module: Optional[Union[BaseDataModule, UnlabeledDataModule]] = None,
+) -> None:
+    """Compute various metrics on predictions csv file."""
+
+    # get keypoint names
+    labels_file = os.path.join(cfg["data"]["data_dir"], cfg["data"]["csv_file"])
+    labels_df = pd.read_csv(labels_file, header=list(cfg["data"]["header_rows"]), index_col=0)
+    keypoint_names = get_keypoint_names(
```

Comment:
if we input `data_module` I think we can get `keypoint_names` from `data_module.dataset.keypoint_names`
see [this line](https://github.com/danbider/lightning-pose/blob/22b6ebe69390db221743b483556242fb24acb2a1/lightning_pose/data/datasets.py#L90)

Reply:
good catch! i'm gonna leave for now though since `data_module` is optional, only used for pca losses. i don't want someone to have to create a data module when computing pixel error say just to get kp names